CONTRIBUTING.md

```diff
@@ -4,9 +4,9 @@ Thank you for helping make our documentation better!
 
 In order to maintain a consistent style and voice throughout our documentation please try to follow these standards and conventions when filing Pull Requests against our documentation.
 
-## Markdown
+## Hugo & Markdown
 
-GitBook (which we use for documentation) uses Markdown as its file format.  
+Hugo (which we use for documentation) uses Markdown as its file format.  
 
 * All files should end in `.md`. They should be all-lower-case and match the title of the page, or in some cases an abbridged version of it.
 * All pages should start with a title, denoted with # (first level header).
```

Comment:
This should be updated for the YAML frontmatter.

Reply:
I'm not sure I know where you're referring to here

Reply:
"Pages should start with a title denoted with #".  That's GitBook-ese.  Hugo uses the YAML frontmatter instead.

Reply:
Ah, okay gotcha